### DISCUSSION TITLE
How to configure routing forwarding？

### DISCUSSION THREAD

**Discussion by yisirman** [SoftEtherVPN/SoftEtherVPN#1934] [General]
When a remote client connects, all traffic will be forwarded from the server by default. How to configure this? Only fixed network segments will be routed and forwarded from the VPN server. No forwarding is required to save VPN server traffic ？

**Comment by officialh1** [comment]
You can probably use Manage Access Lists in the Virtual Hub.  I think more detail about your setup would be helpful. Is this site to site or end user to server, what the hub setting, etc.
